bsp/test/org/jetbrains/bsp/project/resolver/BspResolverLogicTest.scala

```diff
@@ -12,7 +12,7 @@ class BspResolverLogicTest {
   @Test def testCalculateModuleDescriptionsEmptyBaseDir(): Unit = {
 
     val target = new BuildTarget(
-      new BuildTargetIdentifier("ePzqj://jqke:540/n/ius7/jDa/t/z78"),
+      new BuildTargetIdentifier("file:///jqke:540/n/ius7/jDa/t/z78"),
```

Comment:
does this break now with non-`file` URIs? technically any valid URI should be a valid identifier

Reply:
This test should work again in the original form